pkgs/tools/misc/mommy/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, writeText
+, shellspec
+, fetchpatch
+  # usage:
+  # pkgs.mommy.override {
+  #  mommySettings.sweetie = "catgirl";
+  # }
+  #
+  # $ mommy
+  # who's my good catgirl~
+, mommySettings ? null
+}:
+
+let
+  variables = lib.mapAttrs'
+    (name: value: lib.nameValuePair "MOMMY_${lib.toUpper name}" value)
+    mommySettings;
+  configFile = writeText "mommy-config" (lib.toShellVars variables);
+in
+stdenv.mkDerivation rec {
+  pname = "mommy";
+  version = "1.2.3";
+
+  src = fetchFromGitHub {
+    owner = "FWDekker";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-LT21MJg2rF84o2rWKguEP4UUOOu27nNGls95sBYgICw=";
+  };
+
+  patches = [
+    # 🫣 mommy finds your config with an environment variable now~
+    (fetchpatch {
+      url = "https://github.com/FWDekker/mommy/commit/d5785521fe2ce9ec832dbfe20abc483545b1df97.patch";
+      hash = "sha256-GacUjQyvvwRRYKIHHTTAL0mQSPMZbbxacqilBaw807Y=";
+    })
+    # 💜 mommy follows the XDG base directory specification now~
+    (fetchpatch {
+      url = "https://github.com/FWDekker/mommy/commit/71806bc0ace2822ac32dc8dd5bb0b0881d849982.patch";
+      hash = "sha256-Znq3VOgYI7a13Fqyz9nevtrvn7S5Jb2uBc78D0BG8rY=";
+    })
+  ];
+
+  nativeBuildInputs = [ makeWrapper shellspec ];
```

Comment:
Is shellspec only required for tests? It should probably be `nativeCheckInputs` if so. Otherwise this looks good to me!

Reply:
done

TIL [this is a no-op](https://github.com/ckiee/nixpkgs/blob/c9e0fc8e93047e165d9202f30073ffb37c48b5a2/pkgs/stdenv/generic/make-derivation.nix#L227-L231) on standard `stdenv.mkDerivation`s. not 100% sure on that.

Reply:
Checks can still be forced off (e.g. for cross) so this would avoid that dep in that case. But it's admittedly super minor. Thank you for updating it, though!